code/modules/power/lighting.dm

```diff
@@ -395,6 +397,12 @@
 			update(0)
 		flickering = 0
 
+// random flicker
+/obj/machinery/light/process()
+	..()
+	if(prob(3))

```

Comment:
A single `flicker()` call takes on average as much time as its argument - in this case it's 20 seconds.

So, every 2 seconds you have a 3% chance for each and every light on the station to start flickering for 20 seconds.

That means that every light at all times has a `1 - (97%)^20 = 46%` chance to be flickering in any given moment.

Tone that down to 1%.